error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/NonStaticImport.java

```diff
@@ -78,6 +78,7 @@ public final class NonStaticImport extends BugChecker implements CompilationUnit
   static final ImmutableSetMultimap<String, String> NON_STATIC_IMPORT_CANDIDATE_MEMBERS =
       ImmutableSetMultimap.<String, String>builder()
           .put("com.google.common.base.Predicates", "contains")
+          .put("com.google.errorprone.BugCheckerRefactoringTestHelper.TestMode", "TEXT_MATCH")
```

Comment:
It's not like we do want to statically import `AST_MATCH`, so I suppose we can extend `NON_STATIC_IMPORT_CANDIDATE_TYPES` instead.